src/Grpc/test/InteropTests/InteropTests.cs

```diff
@@ -25,7 +25,7 @@ public InteropTests(ITestOutputHelper output)
             _output = output;
         }
 
-        [Fact(Skip= "https://github.com/dotnet/aspnetcore/issues/24902")]
+        [Fact]
```

Comment:
Question only: Should these tests be quarantined for now❔

Reply:
Famous last words: it should just work 😄